test/api/search_controller_test.rb

```diff
@@ -264,24 +277,15 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
   test 'should return preview of bulk download files and total bytes' do
     puts "#{File.basename(__FILE__)}: #{self.method_name}"
 
-    study = Study.find_by(name: "Test Study #{@random_seed}")
+    study = Study.find_by(name: "Testing Study #{@random_seed}")
```

Comment:
is there a reason for this name change?  I like 'Test Study' better as a name (although either is fine)

Reply:
This is the first study created in `db/seeds.rb`, and it is the same study that has the synthetic BQ seeds loaded in.